lib/issuetrackerintegration/githubrestInterface.class.php

```diff
@@ -0,0 +1,366 @@
+<?php
+/**
+ * TestLink Open Source Project - http://testlink.sourceforge.net/ 
+ *
+ * @filesource	githubrestInterface.class.php
+ * @author jlguardi <jlguardi@gmail.com>
+ *
+ * @internal revisions
+ * @since 1.9.16
```

Comment:
this seems to be wrong because this code is going to be added to version 1.9.20-fixed

Reply:
updated (it came from gitlab code)